Files/Views/Settings.xaml.cs

```diff
@@ -43,6 +46,8 @@ protected override void OnNavigatedFrom(NavigationEventArgs eventArgs)
         {
             var currentView = SystemNavigationManager.GetForCurrentView();
             currentView.BackRequested -= OnBackRequested;
+
+            Window.Current.CoreWindow.Dispatcher.AcceleratorKeyActivated -= CoreDispatcher_AcceleratorKeyActivated;
```

Comment:
Shouldn't this be in `CoreDispatcher_AcceleratorKeyActivated`?

Reply:
I used the same logic as for OnBackRequested. It unsubscribes even if we quit using the button. In your proposal, you can only unsubscribe when you exit using the keyboard shortcut.

Reply:
@duke7553 will this work?